Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.5-alpine
+FROM python:3.6-alpine
```

Comment:
Does still work with python 3.5? There are a lot of files that should be change if Python 3.5 is not supported anymore (`Readme`, `setup.py`, etc.)

Reply:
Yes, actually it works even with 3.7. This change was a test that I forgot to undo before commit. I'm gonna revert this change and open an issue to check that we can certainly upgrade to Python 3.7 and maybe use this more updated version.